src/components/Textarea/index.tsx

```diff
@@ -3,9 +3,24 @@ import styled from '@emotion/styled'
 import { useRef } from 'react'
 
 export interface TextAreaProps extends HTMLAttributes<HTMLTextAreaElement> {
+  /** TextArea 컴포넌트 상단의 label 정합니다.
+   * @type string
+   */
   label?: string
+
+  /** TextArea 컴포넌트 하단의 guideMessage 정합니다.
+   * @type string
+   */
```

Comment:
```suggestion
  /** TextArea 컴포넌트 하단의 안내 메시지를 정합니다.
   * @type string
   */
```
props명이어서 한글로 풀어써도 좋을 것 같아용!